core/src/tangram.cpp

```diff
@@ -75,9 +82,11 @@ class Map::Impl {
     TileWorker tileWorker{MAX_WORKERS};
     TileManager tileManager{tileWorker};
     MarkerManager markerManager;
+    FrameBuffer selectionBuffer{256, 256};
```

Comment:
Yes those screenshots you've taken seem to give better results than the debug toggle that was there since you see the actual resolution, we should use that one instead.